packages/moonstone/BodyText/tests/BodyText-specs.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import BodyText from '../BodyText';
+import css from '../BodyText.module.less';
+
+describe('BodyText Specs', () => {
+
+	test('should support multi-line content', () => {
```

Comment:
This doesn't describe the test case correctly. This is more like verifying the default component is `p` tag. Let's update the description text.

Reply:
The description is actually what I wanted to test. Do you have any advice on how the test should be rewritten to be more accurate to the description?

Reply:
I didn't actually know of a "real" way to test this, other than just trusting that if marquee wasn't there (and P was) that the CSS wouldn't clip the additional lines of text.